base-template/contributors-guide.md

```diff
@@ -1,51 +1,250 @@
+The Good Docs Project contains repositories for the templates,website, governance, and more.
+To contribute to the templates, you need to work with the `templates` repo, and the `incubator` repo.
 
-### The `templates` repository
+### The `templates` repository
 
-Energistically visualize excellent technology through mission-critical expertise. Energistically communicate dynamic human capital via backend infrastructures. Continually visualize professional solutions rather than highly efficient relationships. Interactively create real-time models after cross-unit convergence. Professionally parallel task vertical quality vectors through prospective potentialities.
+This is the main repository in the project.
+It contains the templates, and their companion guidance documents.
+To make a small change to an existing template, such as updating some wording, or fixing a typo, propose a change directly within this repository.
 
-### The `incubator` repository
+### The `incubator` repository
 
-Completely maintain technically sound action items after error-free functionalities. Credibly repurpose holistic testing procedures rather than equity invested ideas. Dramatically evolve enterprise methods of empowerment whereas 2.0 intellectual capital. Collaboratively parallel task premium content before customized relationships. Enthusiastically scale multidisciplinary deliverables without corporate e-tailers.
+For bigger projects, including creating a new template, use the `incubator` repository.
+Use this repository to work on changes in collaboration with the rest of the community, and move it into the appropriate location when complete.
 
 ## Setting up your work environment
 
-Globally promote B2C users without client-centered ideas. Uniquely administrate world-class bandwidth before future-proof vortals. Holisticly administrate enterprise e-business for visionary mindshare. Efficiently aggregate robust imperatives for flexible markets. Efficiently disintermediate emerging resources without adaptive technology.
+To work on content within the Good Docs Project, you need to set up your work environment.
 
 ### Selecting a text editor
 
-Continually unleash resource maximizing web services rather than high-payoff convergence. Dynamically revolutionize high standards in innovation whereas process-centric initiatives. Globally seize empowered leadership via quality growth strategies. Dynamically target quality partnerships rather than revolutionary methodologies. Dynamically reinvent superior action items via optimal technology.
+A large range of text editors is available for download.
```

Comment:
I originally had some stuff suggesting vscode and atom as advanced editors, but then took it out because it felt a little wordy. I think that if you have already installed an editor, you probably wouldn't be very interested in this section anyway.

Reply:
All that said, we could append "such as VSCode or Atom" to L48, too :thinking: